src/mixins/state.js

```diff
@@ -110,6 +110,21 @@ export default {
     return this._stateModel.set.apply(this._stateModel, arguments);
   },
 
+
+  /**
+   *  Reset _stateModel to defined defaults
+   *
+   * @public
+   * @method resetState
+   * @returns {Backbone.Model|*} - The _stateModel or the attribute value of the _stateModel
+   */
+
+  resetState() {
+    const defaults = this._stateModel.defaults;
+
+    return this._stateModel.set.apply(this._stateModel, defaults);
```

Comment:
I know we used the apply somewhere.. but I think in this case it isn't apply.. it's just `this._stateModel.set(defaults)`?  but defaults can be a function.. so it needs to be `const defaults = _.result(this._stateModel, 'defaults');`

Definitely needs a test or two.. maybe for when defaults is a hash or a function.. maybe one if it doesn't exist.